src/client.h

```diff
@@ -193,6 +193,7 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	void handleCommand_ActiveObjectRemoveAdd(NetworkPacket* pkt);
 	void handleCommand_ActiveObjectMessages(NetworkPacket* pkt);
 	void handleCommand_Movement(NetworkPacket* pkt);
+	void handleCommand_Fov(NetworkPacket* pkt);
```

Comment:
Wrong pointer placement.

Reply:
Shall I replace all occurrences in that file?